src/main/scenes/svg.tsx

```diff
@@ -212,19 +214,56 @@ export class Svg extends Component<Props, State> {
-              return (
-                <svg
-                  x={element.bounds.x - translationFactor().minX}
-                  y={element.bounds.y - translationFactor().minY}
-                  width={element.bounds.width}
-                  height={element.bounds.height}
-                  key={element.id}
-                  className={element.name ? element.name.replace(/[<>]/g, '') : ''}
-                  fill={element.fillColor || theme.color.background}
-                >
-                  <ElementComponent key={index} element={element} scale={this.props.options?.scale || 1.0} />
-                </svg>
-              );
+              // If the ElementComponent is of type UMLClassifierComponent, create an array of all members (attributes and methods)
+              if (ElementComponent === UMLClassifierComponent) {
+                const members = elements.filter((member) => member.owner === element.id);
+                return (
+                  <svg
+                    x={element.bounds.x}
+                    y={element.bounds.y}
+                    width={element.bounds.width}
+                    height={element.bounds.height}
+                    key={element.id}
+                    className={element.name ? element.name.replace(/[<>]/g, '') : ''}
+                    fill={element.fillColor || theme.color.background}
+                  >
+                    <ElementComponent key={index} element={element} scale={this.props.options?.scale || 1.0}>
+                      {members.map((memberElement, memberIndex) => {
+                        // Nest the members within the UMLClassifierComponent, so the divider path and rectangle get rendered afterwards
+                        const MemberElementComponent = Components[memberElement.type as UMLElementType];
+                        return (
+                          <svg
+                            x={0}
+                            y={memberElement.bounds.y - element.bounds.y}
+                            width={memberElement.bounds.width}
+                            height={memberElement.bounds.height}
+                            key={memberElement.id}
+                            className={memberElement.name ? memberElement.name.replace(/[<>]/g, '') : ''}
+                            fill={memberElement.fillColor || theme.color.background}
+                          >
+                            <MemberElementComponent key={memberIndex} element={memberElement} scale={this.props.options?.scale || 1.0} />
+                          </svg>
+                        );
+                      })}
+                    </ElementComponent>
+                  </svg>
+                );
+                // Render all other UMLElements and UMLRelationships normally, as they dont have issues when rendering to SVG
+              } else if (ElementComponent !== UMLClassifierComponent && ElementComponent !== UMLClassifierMemberComponent) {
```

Comment:
I would move the comment into the `else`
```suggestion
              } else if (ElementComponent !== UMLClassifierComponent && ElementComponent !== UMLClassifierMemberComponent) {
                  // Render all other UMLElements and UMLRelationships normally, as they dont have issues when rendering to SVG
```